Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM cidasdpdasartip.cr.usgs.gov:8447/aqcu/rserve:latest
+
+ENV GSPLOT_VERSION_DEFAULT=0.8.1
+
+COPY inst/extdata/installPackages.R /tmp/install/installPackages.R
+
+# Get GSPlot dependency installs
+RUN mkdir -p /tmp/install/gsplot_description_dir && \
+  wget -O /tmp/install/gsplot_description_dir/DESCRIPTION https://raw.githubusercontent.com/USGS-R/gsplot/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}/DESCRIPTION
+
+# Install GSplot
+RUN mkdir ${RSERVE_HOME}/R_libs && \
+  mkdir ${RSERVE_HOME}/work && \
+  cd /tmp/install/gsplot_description_dir && \
+  Rscript /tmp/install/installPackages.R && \
+  Rscript -e "library(devtools);install_url('https://github.com/USGS-R/gsplot/archive/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}.zip', dependencies = F)"
+
+# Install Repgen Dependencies
+COPY DESCRIPTION /tmp/install/repgen/DESCRIPTION
+RUN cd /tmp/install/repgen && Rscript /tmp/install/installPackages.R
+
+# Copy Repgen Sources
+COPY inst /tmp/install/repgen/inst
+COPY man /tmp/install/repgen/man
+COPY R /tmp/install/repgen/R
+COPY tests /tmp/install/repgen/test
+COPY .Rbuildignore /tmp/install/repgen/.Rbuildignore
+COPY LICENSE /tmp/install/repgen/LICENSE
+COPY NAMESPACE /tmp/install/repgen/NAMESPACE
+
+# Build and install Repgen Package
+RUN cd /tmp/install/ && R CMD build repgen && \
+  mv /tmp/install/repgen_*.tar.gz /tmp/install/repgen.tar.gz && \
+  Rscript -e "install.packages(\"/tmp/install/repgen.tar.gz\", repos = NULL, type=\"source\")" && \
```

Comment:
might need to make sure to do the dependencies=F again here, otherwise we'll be getting uncontrolled new dependency versions?

Reply:
Unless I'm not understanding how things have changed from the other Dockerfile

Reply:
Ah, I wasn't sure if that was available for the `install.packages()` command (we were using `install_gitub()` before), but I can try adding it back!

Reply:
Ah, you might be good, because https://www.rdocumentation.org/packages/utils/versions/3.5.3/topics/install.packages says Not used if repos = NULL. 

so, I guess just make sure that it has the locked-down versions we're hoping for?

Reply:
you can get into R on the container by typing 'R' as the ec2-user and then hit enter, and then do a packageVersion("packagename")